microbench/src/main/java/io/netty/handler/codec/CodecOutputListBenchmark.java

```diff
@@ -54,16 +45,19 @@ public void destroy() {
 
     @Benchmark
     public void codecOutList() {
+        codecOutputList = CodecOutputList.newInstance();
         benchmarkAddAndClear(codecOutputList, elements);
     }
 
     @Benchmark
     public void recyclableArrayList() {
+        recycleableArrayList = RecyclableArrayList.newInstance(16);
         benchmarkAddAndClear(recycleableArrayList, elements);
```

Comment:
Shouldn't we call `recycleableArrayList.release()` here  and remove it from the `@TearDown` ?